src/client/pages/OfferNew/index.tsx

```diff
@@ -46,7 +46,11 @@ const getQuoteIdsFromBundleVariant = (bundleVariant: QuoteBundleVariant) =>
   bundleVariant.bundle.quotes.map((quote) => quote.id)
 
 export const OfferNew: React.FC = () => {
-  const { path: localePath, isoLocale, phoneNumber } = useCurrentLocale()
+  const {
+    path: localePath,
+    isoLocale,
+    callCenter: phoneNumber,
```

Comment:
Could this be passed as `callcenter`?

Reply:
Agree.

Reply:
Tried to keep changes minimal but I think you guys are right. I will fix